MAINTAINERS.md

```diff
@@ -28,15 +28,16 @@ We also have some nice souls that help triaging issues and PRs. See [here](https
 
 Full list of triage persons is displayed below:
 
-| Name          | Slack         | GitHub                                     | Company |
-|---------------|---------------|--------------------------------------------|---------|
-| Adrien Fillon | `@Adrien F`   | [@adrien-f](https://github.com/adrien-f)   |         |
-| Ian Billett   | `@billett`    | [@bill3tt](https://github.com/bill3tt)     | Red Hat |
-| Martin Chodur | `@FUSAKLA`    | [@fusakla](https://github.com/fusakla)     |         |
-| Michael Dai   | `@jojohappy`  | [@jojohappy](https://github.com/jojohappy) |         |
-| Xiang Dai     | `@daixiang0`  | [@daixiang0](https://github.com/daixiang0) |         |
-| Jimmie Han    | `@hanjm`      | [@hanjm](https://github.com/hanjm)         | Tencent |
-| Matej Gera    | `@Matej Gera` | [@matej-g](https://github.com/matej-g)     | Red Hat |
+| Name            | Slack              | GitHub                                       | Company |
+|-----------------|--------------------|----------------------------------------------|---------|
+| Adrien Fillon   | `@Adrien F`        | [@adrien-f](https://github.com/adrien-f)     |         |
+| Ian Billett     | `@billett`         | [@bill3tt](https://github.com/bill3tt)       | Red Hat |
+| Martin Chodur   | `@FUSAKLA`         | [@fusakla](https://github.com/fusakla)       |         |
+| Michael Dai     | `@jojohappy`       | [@jojohappy](https://github.com/jojohappy)   |         |
+| Xiang Dai       | `@daixiang0`       | [@daixiang0](https://github.com/daixiang0)   |         |
+| Jimmie Han      | `@hanjm`           | [@hanjm](https://github.com/hanjm)           | Tencent |
+| Matej Gera      | `@Matej Gera`      | [@matej-g](https://github.com/matej-g)       | Red Hat |
+| Filip Petkovski | `@Filip Petkovski` | [@fpetkovski](https://github.com/fpetkovski) |         |
```

Comment:
Is it okay to add your company name? Not sure why many team members don't have a company name here.

Reply:
I would prefer to omit it for now due to temporary internal policy that we have in place. I'll add it in several months once the policy has expired :)